importer.js

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+import nodePath from 'path';
+import csvjson from 'csvjson';
+import { promisify } from 'util';
+const readFileAsync = promisify(fs.readFile);
+const readDirAsync = promisify(fs.readdir);
+
+class Importer {
+    constructor(dirWatcher) {
+        dirWatcher.on('changed', async (path) => {
+            /**
+             * Watching for file changes Syncrhonously
+             */
+            let syncData = this.importSync(path);
+            console.log(result)
+
+            /**
+             * Asynchronously watching the file changes
+             */
+            let asyncData = await this.import(path);
+            console.log(asyncData); 
+        });
+    }
+
+    async import(path) {
+        try {
+            let jsonData = [], newFileData = [];
+            let fileNames = await readDirAsync(path);
+            for (const fileName of fileNames) {
+                let data = await readFileAsync(nodePath.join(__dirname, path, fileName), { encoding: 'utf8' });
```

Comment:
if there is no specific need to change the variable value `const` is preferable than `let`

Reply:
Added const over let.